### DISCUSSION TITLE
Should OVAL6 be in a separate repository or branch?

### DISCUSSION THREAD

**Discussion by vanderpol** [OVAL-Community/OVAL#160] [Q&A] [answered]
@OVAL-Community/oval-board-members 
During the 09-26-2024 meeting, we were not able to come to a clear consensus as to to proceed with OVAL6.  There are pros and cons to each.  

Due to the limited timeframe to complete all of the OVAL updates, this discussion will end on 10-01-2024, and vote will occur on 10-02-2004 and end on 10-04-2024.

**Comment by DeafBDoor** [comment, marked as answer]
Vote for the same repository, different branches. Leave the master/main branch clean and use different branches for different language versions. Tag the releases and noteworthy development commits and use the github release mechanism.

**Reply by vanderpol** [reply]
After some more thought on the topic I'm in agreement with this as well, I spent the last day getting more familiar with the github release process, and I'm wondering if we should move some of our OVAL board based documentation to a different repository?  Create an OVAL Board Artifacts Repo or something?  When you do the github build, the 'source code.zip' file contains the entire repository, I may start another discussion on Repo maintenance.

**Reply by DeafBDoor** [reply]
Well, there is the option to create a pipeline script which cleans/processes the artifacts and send them to the release.

**Comment by vanderpol** [comment]
Closing discussion, OVAL 6 will remain within the OVAL repository.  OVAL 5.12 will be a branch, and OVAL6 will get it's own develop branch
